src/status_im2/contexts/wallet/common/screen_base/create_or_edit_account/view.cljs

```diff
@@ -45,12 +46,21 @@
          :i/reaction]]
        [quo/title-input
         {:placeholder     (i18n/label :t/account-name-input-placeholder)
-         :max-length      24
+         :max-length      20
          :blur?           true
          :default-value   account-name
          :on-change-text  on-change-name
-         :container-style style/title-input-container
-         :return-key-type :done}]
+         :container-style (style/title-input-container error-message)
+         :return-key-type :done
+         :on-focus        on-focus
+         :on-blur         on-blur}]
+       (when error-message
```

Comment:
I think we should ask designers to revisit this and use a neutral info message when there's no error - similar to create profile page. wdyt? I don't like when the content jumps like this 🙃

Reply:
I'm sorry, Jamie. I can't understand the part where using a neutral info message when there's no error. 

I added this `error-message` for display name validation error messages which I will do later.

<img width="800" src="https://github.com/status-im/status-mobile/assets/19339952/32899068-bfe7-42fa-ba6c-658552b3b90e">

Also, this would be needed in the creation of account flow.

Reply:
I asked this in design channel. 
Yes definitely we need this in create account/ add watch account flows too. But imo we should leave validation until anyway, if we're adding some for UX we should just add the bare minimum in initial pr's to not block the work going in

Reply:
Apologies, I understand the context now. 
Yes, the screen would jump if there is an error. I like the idea of showing a neutral message 👍

Reply:
Sure 👍